Documentation/HowToUseETW.md

```diff
@@ -0,0 +1,65 @@
+# Collecting and Analyzing ETW Traces 
+
+## Supported
+ETW logs are available on Windows platform only currently.
+
+## Collect ETW logs for WCF Server
+* If you use your own WCF Server, it's recommended to follow https://msdn.microsoft.com/en-us/library/ms733025(v=vs.110).aspx.
+* If you run outerloop tests that use bridge, it's recommended to follow "How to Collect ETW logs uses logman" below.
+
```

Comment:
Nit: no need to mention the bridge or outerloop here (we don't yet document those well).  Though you might say something like "If you run the scenario tests, a locally-hosted WCF service will be started automatically, and it is recommended..."